src/ui/application-window.vala

```diff
@@ -330,12 +332,30 @@ namespace Peek.Ui {
         if (get_window_width () < SMALL_WINDOW_SIZE) {
           GtkHelper.hide_button_label (record_button);
           GtkHelper.hide_button_label (stop_button);
+          //Set the scale of shortcut_label
+          Pango.AttrList attrs = new Pango.AttrList (); 
+          attrs.insert (Pango.attr_scale_new (Pango.Scale.SMALL)); 
+          shortcut_label.attributes = attrs;
         } else {
           GtkHelper.show_button_label (record_button);
           GtkHelper.show_button_label (stop_button);
+          //Set the scale of shortcut_label
+          Pango.AttrList attrs = new Pango.AttrList (); 
+          attrs.insert (Pango.attr_scale_new (Pango.Scale.LARGE)); 
+          shortcut_label.attributes = attrs;
         }
 
         if (!is_recording) {
+          var shortcut =Application.get_app_settings();
```

Comment:
Please make sure to have a space before and after `=`. Also Peek commonly uses a space before function parantheses. That's not a style I normally use, but most other Vala apps I saw used that, and I thought it would make sense to follow commonly used coding style ;)

Reply:
oops.. that is one thing I really need to work on